python/ambassador-diag/src/ambassador_diag/diagd.py

```diff
@@ -1840,12 +1847,21 @@ def _load_ir(
         service_count = len(app.ir.services)
 
         self._respond(
-            rqueue, 200, "configuration updated (%s) from snapshot %s" % (config_type, snapshot)
+            rqueue,
+            200,
+            "configuration updated (%s) from snapshot %s" % (config_type, snapshot),
         )
 
         self.logger.info(
             "configuration updated (%s) from snapshot %s (S%d L%d G%d C%d)"
-            % (config_type, snapshot, service_count, listener_count, group_count, cluster_count)
+            % (
+                config_type,
+                snapshot,
+                service_count,
+                listener_count,
+                group_count,
+                cluster_count,
+            )
```

Comment:
Because `ruff` preserves behavior from `black`, and therefore because of the trailing comma rule in black [[ref](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#c-skip-magic-trailing-comma)], at least I assume?